contributing/core/developing.md

```diff
@@ -7,9 +7,9 @@
 To develop locally:
 
 1. Install the [GitHub CLI](https://github.com/cli/cli#installation).
-1. Clone the Next.js repository:
+1. Clone the Next.js repository (downloads only the last commit for faster clone):
    ```
-   gh repo clone vercel/next.js
+   gh repo clone vercel/next.js --depth=1 --branch canary --single-branch
```

Comment:
Can you verify what happens if you e.g. clone the last ~500 to 1000 commits instead? Doing a 1 depth clone will basically cause gitlens / git blame to not be the best.

Reply:
Cloning 2000 is since 16 June 2022. It will download ~60 MB.
Gitblame won't work well anyway since it's most of the code that I edit is over a year old, but a lot of contributors are fixing bugs that were introduced recently, so It could still help.

Reply:
Coll 3000 is still 65MB. 4000 starts to be slow.

Reply:
https://github.com/vercel/next.js/pull/44181